templates.go

```diff
@@ -1049,7 +1467,6 @@ coreos:
       ExecStartPre=/usr/bin/docker pull $IMAGE
       ExecStartPre=-/usr/bin/docker stop -t 10 $NAME
       ExecStartPre=-/usr/bin/docker rm -f $NAME
-      ExecStartPre=/bin/sh -c "while ! curl --output /dev/null --silent --head --fail --cacert /etc/kubernetes/ssl/apiserver-ca.pem --cert /etc/kubernetes/ssl/apiserver-crt.pem --key /etc/kubernetes/ssl/apiserver-key.pem https://{{.Cluster.Kubernetes.API.Domain}}; do sleep 1 && echo 'Waiting for master'; done"
```

Comment:
This looks like it doesn't belong into this PR. 

Why did you remove this?

Reply:
We actually removed this from all units long time ago, seems we forgot some units. I needed to remove this for RBAC/PSP as this would prevent stuff from coming up. We are not running into race conditions without it, the components just retry, so it's not needed.